apps/nextjs-website/stories/Buttons/dark.stories.ts

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { CtaButtons } from '@react-components/components/common/Common';
+import { CtaButtonProps } from '@react-components/types/common/Common.types';
+
+interface CtaButtonsProps {
+  readonly ctaButtons: ReadonlyArray<CtaButtonProps | JSX.Element>;
+  readonly theme?: 'dark' | 'light';
+  readonly disableRipple?: boolean;
+}
+
+const meta: Meta<CtaButtonsProps> = {
+  title: 'General/Buttons/Dark',
+  component: CtaButtons,
+  parameters: {
+    layout: 'centered'
+  },
+  argTypes: {
+    theme: {
+      control: {
+        type: 'select',
+        options: ['dark', 'light'],
+      },
+    },
+    disableRipple: {
+      control: 'boolean',
+    },
+    ctaButtons: {
+      control: 'object',
+    },
+  },
```

Comment:
Since this is all the same for light and dark I would move this to `buttonsCommons.tsx` as well